jetson/percep/pcl.cpp

```diff
@@ -185,177 +185,81 @@ void PCL::FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices,
 
 }
 
+/* --- Get Angle --- */
+//This function finds the angle off the x axis the
```

Comment:
What is the x axis in your case? The ground? Could you please clarify this in the comment so it's never ambiguous.